Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderNode.spec.json

```
@@ -2248,6 +2248,10 @@
                             "$ref": "#/components/schemas/TopicImage"
                         }
                     },
+                    "custom": {
+                       "key" : "string",
+                       "value" : "string"
+                    }
```

This only allows for a single piece of custom metadata and we allow multiple.

```suggestion
                    "custom": {
                         "type": "object",
                         "description" : "A dictionary containing arbitrary, user-provided metadata key/value pairs.\n\nThese values are authored using the `@CustomMetadata(key:value:)` directive.",
                         "additionalProperties": {
                             "type": "string"
                     },
```

---

took care of it. please take a look now.